modules/ROOT/pages/index.adoc

```diff
@@ -1,4 +1,4 @@
-= MuleSoft Documentation
+= Welcome to the Anypoint Platform
```

Comment:
```suggestion
= Welcome to Anypoint Platform
```

Reply:
I did not make comments about the legacy text hidden below, but I could copyedit it at some point, when it is more accessible to editing.